en/README.md

```diff
@@ -0,0 +1,1138 @@
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchHistory(method)
+
+This procedure request to get history to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched history using a putHistory procedure. At that time, need to pass fetchId of fetchHistory procedure params to putHistory procedure.
+
+Assume to get matched history that from beginTime till endTime.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostId    |String255|M|-|Host Id that the item belongs.|
+|itemId    |String255|M|-|Item Id of history.|
+|beginTime |TimeStamp|M|-|Starting point of history.|
+|endTime   |TimeStamp|M|-|End point of history.|
+|fetchId|String255    |M|-|Use in putHistory procedures. It need to identify the putHistory corresponding to the fetchHistory.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "fetchId": 1,
+    "beginTime": "20150323151300",
+    "valueType": "INTERGER",
+    "itemId": 1,
+    "hostId": "1"
+  },
+  "method": "fetchHistory",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchTriggers(method)
+
+This procedure request to get triggers to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched triggers using a putTriggers procedure. At that time, need to pass fetchId and hostIds of fetchTriggers procedure params to putTriggers procedure.
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|hostIds|String255 array|O|-|Get triggers of to designate a hosts. If does not designate hostIds, all trigger are targeted.|
+|fetchId|String255    |M|-|Use in putTriggers procedures. It need to identify the putTriggers corresponding to the fetchTriggers.|
+
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "fetchTriggers",
+  "params": {
+    "hostIds": [
+      "1",
+      "2"
+    ],
+    "fetchId": "1"
+  },
+  "id": 1
+}
+```
+
+***Result(result)***
+
+Return result as result object value whether request accept to Hatohol server. Look at the [[Values](#user-content-putresult)].
+
+```json
+{
+  "id": 1,
+  "result": "SUCCESS",
+  "jsonrpc": "2.0"
+}
+```
+
+### fetchEvents(method)
+
+This procedure request to get designate number of events that sorted by ascending or descending order to HAP. HAP must return success or failure of to accept a that request. After that, Send to matched event using a putEvents procedure. At that time, need to pass fetchId of fetchEvents procedure params to putEvents procedure.
+
+Can designate "count" value till 1000. If want to request over the event number 1000, separate request to more than once.
```

Comment:
> If want to request over the event number 1000, separate request to more than once.

If Hatohol server want to request over 1000 events, split the request into two or more to make count field less than 1000.